deploy/helm/charts/values.yaml

```diff
@@ -18,31 +18,138 @@ rbac:
 imagePullSecrets:
 # - name: "image-pull-secret"
 
+openebs-ndm:
```

Comment:
This is not required here, It will be picked up from the dependent chart

Reply:
Sorry, didn't understand. Are you talking about that line, or the parameters in general?

Reply:
The parameters in general. Since we already have all of them in the values.yaml in the the openebs-ndm dependent chart

Reply:
Removed all attributes of the ndm chart from the parent chart's values.yaml